src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -0,0 +1,82 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoNumbers {
+
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
+    private static final int LOTTO_NUMBERS_LENGTH = 6;
+
+    private static final String DELIMITER = ",";
+
+    private static final List<LottoNumber> allLottoNumber;
+
+    static {
+        allLottoNumber = initializeAllLottoNumber();
+    }
+
+    private List<LottoNumber> lottoNumbers;
+
+    private static List<LottoNumber> initializeAllLottoNumber() {
+        List<LottoNumber> lottoNumbers = new ArrayList<>();
+        for (int i = MIN_LOTTO_NUMBER; i <= MAX_LOTTO_NUMBER; i++) {
+            lottoNumbers.add(new LottoNumber(i));
+        }
+        return lottoNumbers;
+    }
+
+    public LottoNumbers(String stringLottoNumbers) {
+        this.lottoNumbers = Arrays.stream(stringLottoNumbers.split(DELIMITER))
+                                  .map(number -> new LottoNumber(Integer.parseInt(number)))
+                                  .collect(Collectors.toList());
```

Comment:
```suggestion
        this(Arrays.stream(stringLottoNumbers.split(DELIMITER))
                                  .map(number -> new LottoNumber(Integer.parseInt(number)))
                                  .collect(Collectors.toList()));
```

인스턴스 변수에 값을 초기화하는 것은 주생성자를 통해서 구현할 것을 추천